File: WebApi/WebApi/Model/Todo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace WebApi.Model
{
    public class Todo
    {
        [Key]
        public int TodoId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public bool IsDone { get; set; }
        public Todo Parent { get; set; }
    }
}

File: WebApp/app/model/todo.js
class Todo {
  constructor(personId, todoId, title, description, parentId) {
    this.personId = personId;
    this.todoId = todoId;
    this.title = title;
    this.isDone = false;
    this.description = description;
    this.parentId = parentId;
  }
}
